### PR TITLE
Fix/missing copyright headers

### PR DESCRIPTION
**Pull request by mreso**
# What does this PR do?
This PR adds some missing copyright headers and a script to automate the process for potential pre-commit integration

Fixes # (issue)

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/facebookresearch/llama-recipes/blob/main/CONTRIBUTING.md#pull-requests),
      Pull Request section?

Thanks for contributing 🎉!

### CODE REVIEW

**Review by HamidShojanazeri — APPROVED** [review]
Thanks @mreso for the PR.

File: eval/open_llm_leaderboard/hellaswag_utils.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```
Comment (HamidShojanazeri) [inline eval/open_llm_leaderboard/hellaswag_utils.py:1]: can we pls remove it from this files, its mostly coming from eval harness.

Comment (mreso) [reply eval/open_llm_leaderboard/hellaswag_utils.py]: Of course, the harder part will be to tell Open Source Bot that this is correct :-D

Comment (mreso) [reply eval/open_llm_leaderboard/hellaswag_utils.py]: Actually not that hard...the file should be excluded by Open Source Bot now

**Pull request merged**